### DISCUSSION TITLE
Automatic return to the corrsponding line after terminal input

### DISCUSSION THREAD

**Discussion by ghost** [mfussenegger/nvim-dap#530] [Q&A]
As the title shows , When I debug a file which needs some input , I need to focus on the terminal , press `i` , input something ， then exit insert mode and return to the code window. So I wanna know if there is a way to make the cursor automatically return to the corresponding line after I input.

**Comment by litoj** [comment]
How do you pass input to a program? When I open the repl window, it prints the program output but doesn't let me interact with it.
